src/server.jl

```diff
@@ -105,6 +107,54 @@ function append_slash(url_str::AbstractString)
     return string(endswith(uri.path, "/") ? uri : merge(uri; path = uri.path * "/"))
 end
 
+"""
+    get_dir_list(target::AbstractString) -> index_page::AbstractString
+
+Get dir list of current path. generate a index page html.
+target is current dir path.
+"""
+function get_dir_list(target::AbstractString)
+    path = joinpath(abspath(CONTENT_DIR[]), target[2:end])
```

Comment:
why `target[2:end]`? maybe this should be ``lstrip(target, [`/`, `\\`, '.'])`` or something like that?